spec/setup.js

```diff
@@ -107,17 +107,19 @@ const runTest = () => {
     __dirname, '..', 'node_modules', '.bin', 'jasmine');
 
   return new Promise((resolve, reject) => {
-    exec('"' + jasminePath + '"', (error, stdout, stderr) => {
-      if (error) {
-        console.warn(error);
-      }
-      if (stderr) {
-        console.log(stderr);
+    exec(
+      'nyc --reporter=lcov --reporter=text "' + jasminePath + '"',
```

Comment:
Is there a way to throw an error here when coverage is not 100%?

Reply:
I think so https://www.npmjs.com/package/nyc#common-configuration-options

Reply:
Let us do that then.

Reply:
Done.

Reply:
I decided to use a `.nyc` config file since the configuration got larger.